frontend/src/components/widgets/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,255 @@
+    })
+
+    it("should render tick bar with min and max", () => {
+      // @ts-ignore
+      const thumbValue = wrapper
+        .find(UISlider)
+        .prop("overrides")
+        // @ts-ignore
+        .TickBar()
+
+      const thumbValueWrapper = shallow(thumbValue)
+
+      expect(thumbValueWrapper.find(".tickBarMin").text()).toBe("0")
+      expect(thumbValueWrapper.find(".tickBarMax").text()).toBe("10")
+    })
+
+    it("should contain sliderOverrides", () => {
+      Object.keys(sliderOverrides).forEach(property => {
+        expect(wrapper.find(UISlider).prop("overrides")).toHaveProperty(
+          property
+        )
+      })
+    })
+  })
+
+  describe("Single value", () => {
+    it("renders without crashing", () => {
+      const props = getProps()
+      const wrapper = shallow(<Slider {...props} />)
+
+      expect(wrapper).toBeDefined()
+    })
+
+    it("should have a correct value", () => {
+      const props = getProps()
+      const wrapper = shallow(<Slider {...props} />)
+      const UISliderWrapper = wrapper.find(UISlider)
+      const propValue = UISliderWrapper.prop("value")
+
+      expect(propValue).toStrictEqual(props.element.get("default").toJS())
+      expect(propValue[0]).toBeGreaterThanOrEqual(props.element.get("min"))
+      expect(propValue[0]).toBeLessThan(props.element.get("max"))
+    })
+
+    it("should handle value changes", async () => {
+      const props = getProps()
+      const wrapper = shallow(<Slider {...props} />)
+
+      // @ts-ignore
+      wrapper.find(UISlider).prop("onChange")({
+        value: [10],
+      })
+
+      await timeout(200)
+
+      expect(props.widgetMgr.setFloatArrayValue).toHaveBeenCalledWith(
+        props.element.get("id"),
+        [10],
+        { fromUi: true }
+      )
+      expect(wrapper.find(UISlider).prop("value")).toStrictEqual([10])
+    })
+  })
+
+  describe("Range value", () => {
+    it("renders without crashing", () => {
+      const props = getProps({
+        default: [1, 9],
+      })
+      const wrapper = shallow(<Slider {...props} />)
+
+      expect(wrapper).toBeDefined()
+    })
+
+    it("should have a correct value", () => {
+      const props = getProps({
+        default: [1, 9],
+      })
+      const wrapper = shallow(<Slider {...props} />)
+      const UISliderWrapper = wrapper.find(UISlider)
+      const propValue = UISliderWrapper.prop("value")
+
+      expect(propValue).toStrictEqual(props.element.get("default").toJS())
+
+      propValue.forEach(value => {
+        expect(value).toBeGreaterThanOrEqual(props.element.get("min"))
+        expect(value).toBeLessThan(props.element.get("max"))
+      })
+    })
+
+    describe("value should be within bounds", () => {
+      const props = getProps({
+        default: [1, 9],
+      })
+      const wrapper = shallow(<Slider {...props} />)
+
+      it("start > end", () => {
+        // @ts-ignore
+        wrapper.find(UISlider).prop("onChange")({
+          value: [11, 10],
```

Comment:
is it even possible for a user to set this in the UI?

Reply:
Yes, with a range slider you can move the start or the end